### DISCUSSION TITLE
How to keep field empty when metadata provider has data?

### DISCUSSION THREAD

**Discussion by SenorSmartyPants** [advplyr/audiobookshelf#2989] [Q&A]
I delete certain fields when I tag outside of ABS. Subtitle is the main one, for books that have "A Novel" or "Series X, Book 3". But if Audible has that field (which it will because that's where I'm initially scraping from) then it gets filed in again. 

Is there a way to tell ABS not to import certain fields from a metadata provider?

**Comment by nichwall** [comment]
If using the normal match tab, you can uncheck fields and those fields are remembered when using the match tab again (including for other books). Not sure if these checkboxes are respected for Quick Match, but you can change the library settings to not match books that already have an ASIN when doing a quick match to prevent matching a book again if doing Quick Match on a library or something.

![image](https://github.com/advplyr/audiobookshelf/assets/5686638/b267d732-d675-4db9-b45a-15609a37578e)

![image](https://github.com/advplyr/audiobookshelf/assets/5686638/2870008a-dd7e-4f6e-9bb0-37a543b269a0)
